### DISCUSSION TITLE
How can I improve SAEM estimation sampling most efficiently?

### DISCUSSION THREAD

**Discussion by billdenney** [nlmixr2/nlmixr2#142] [Q&A] [answered]
I have a model that I think is the correct model for a dataset, but when estimated with SAEM, the `traceplot()` shows slowly changing parameters that end up being highly correlated and also end up having very large confidence intervals.

I have a few other things that I plan to try for this model to change its model structure, but I'm also curious if there is standard advice about what parameters to change in `saemControl()` to improve the sampling?  My guess is that increasing `nBurn` is the most common answer to this question.  But, I'm less certain about where `nmc` and `nu` come in and if they are better for improving the expectation maximization step efficiency in number of iterations required for good convergence (as illustrated by the `traceplot()`).  I'm also not sure if I'm missing another, good parameter to tweak.

**Comment by mattfidler** [comment, marked as answer]
- `nmc` increases the number of markov chains during the `SA` phase, which means more of the parameter space is explored.
- as the `nu` values increase, it increases the accuracy of the `mcmc` likelihood approximation (Discussed http://lixoft.com/wp-content/uploads/2016/03/monolixMethodology.pdf in section 1.1.2)

Both of these are about exploring the parameter space a bit better
